spec/indexers/descriptive_metadata/subject_geographic_spec.rb

```diff
@@ -655,7 +659,7 @@
       end
 
       it 'drops the punctuation before deduping' do
-        expect(doc).to include('sw_subject_geographic_ssim' => ['Europe'])
+        expect(doc).to include('sw_subject_geographic_ssim' => ['Europe', 'Europe'])
```

Comment:
ew - it includes it twice??  If so, should we dedup with some easy call to some ruby method?